modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelfeaturegeneration/v0.1.0.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/emodelfeaturegeneration/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "this:EmodelFeatureGenerationShape",
+      "@type": "sh:NodeShape",
+      "label": "Trace generation shape definition",
+      "editorialNote": "What context information should be added in this generation ?",
+      "targetClass": "nsg:EmodelFeatureGeneration",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/nexus/provsh/generation/v0.1.0/shapes/GenerationShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:activity",
+              "name": "Activity",
+              "description": "Points at feature experiment activity that generated trace",
```

Comment:
Does it really point at the activity that generated the **Trace**?  Or rather the activity that generated the  **EModelFeature**?

Reply:
EModelFeature I think